living/Korean.md

```diff
@@ -0,0 +1,89 @@
+# Web Language: Korean
+
+Additional names:
+- 한국어
+- 
+
+**News:**
```

Comment:
Please avoid changing the text formatting in these documents (these headings should not be bolded)